code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -248,6 +263,7 @@ GLOBAL_LIST_INIT(sandbag_recipes, list (
 
 /obj/item/stack/sheet/mineral/plasma
 	name = "solid plasma"
+	desc = "Beautiful pure purple crystals, ready to burst into flames if a naked flame touches them..."
```

Comment:
```suggestion
	desc = "Beautiful pure purple crystals, ready to ignite if a naked flame touches them..."
```
Try not to repeat something immediately after using it in a sentence
"Burst into flames if a naked flame" double flame, makes the sentence seem sloppy